app/services/vot/parser.rb

```diff
@@ -0,0 +1,75 @@
+module Vot
+  class Parser
+    class ParseException < StandardError; end
+    SUPPORTED_COMPONENT_VALUE_REQUIREMENTS = [
+      :aal2?,
+      :phishing_resistant?,
+      :hspd12?,
+      :identity_proofing?,
+      :biometric_comparison?,
+    ].freeze
+    Result = Data.define(:vector_of_trust, *SUPPORTED_COMPONENT_VALUE_REQUIREMENTS)
```

Comment:
now we can just inline these right?

```suggestion
    Result = Data.define(
      :vector_of_trust,
      :aal2?,
      :phishing_resistant?,
      :hspd12?,
      :identity_proofing?,
      :biometric_comparison?,
    )
```

Reply:
I have been a little bit conflicted on this one. We definitely want predicates on the result. I'm not sure where we actually start applying the `?`. Let me try this one and see how it looks.